Dfe.Data.SearchPrototype/Web/Tests/ViewTests/SearchPagePresentationTests.cs

```diff
@@ -30,7 +28,7 @@ public async Task Search_WithNoEstablishmentResultsAndNoFacets_ShowsNoResultsTex
             .ReturnsAsync(useCaseResponse);
 
         // act
-        var resultsPage = await _context.OpenAsync($"{homeUri}?searchKeyword=anything");
+        var resultsPage = await _context.OpenAsync($"{homeUri}?searchKeyWord=anything");
```

Comment:
keyword is one word - look it up if you don't believe me :)

Reply:
no idea why we changed it , done